### ISSUE TITLE
Some files don't play with audio

### ISSUE THREAD

**Issue by ggarra13** [darbyjohnston/DJV#417]
I am trying to load the avengers-endgame-trailer-2_h1080.mov and I get:

RtApiAlsa::getDeviceInfo: snd_pcm_open error for device (hw:0,0), Device or resource busy.

on Linux Ubuntu 20.04 LTS.

**Comment by darbyjohnston** [comment]
Hi, thanks for letting me know; you mentioned some files don't play, do you have others that work OK?

**Comment by ggarra13** [comment]
Yes.  But let me rephrase the title of the thread.  Sometimes videos play without audio (printing that same message).  The videos I have that fail are not always the same.  Sometimes they play just fine and sometimes they fail to play with audio.  This is true either from command-line or from the gui.  
This is with the binary that was prebuilt at sourceforge ( I did not try compiling my own DJV ).
